src/main/java/jnr/constants/platform/darwin/Errno.java

```diff
@@ -12,7 +12,7 @@
 ENOEXEC(8L),
 EBADF(9L),
 ECHILD(10L),
-EDEADLK(11L),
```

Comment:
So far as I can tell all these constants do line up numerically unless the other PR...

Reply:
#41 removes the sorting and just goes on the .rb order. EDEADLK moved down below with the same value.